config/clients/js/template/client.mustache

```diff
@@ -0,0 +1,438 @@
+  async readAuthorizationModels(options: ClientRequestOpts & PaginationOptions = {}): PromiseResult<ReadAuthorizationModelsResponse> {
+    return this.api.readAuthorizationModels(options.pageSize, options.continuationToken, options);
+  }
+
+  /**
+   * WriteAuthorizationModel - Create a new version of the authorization model
+   * @param body
+   * @param options
+   */
+  async writeAuthorizationModel(body: WriteAuthorizationModelRequest, options: ClientRequestOpts = {}): PromiseResult<WriteAuthorizationModelResponse> {
+    return this.api.writeAuthorizationModel(body, options);
+  }
+
+  /**
+   * ReadAuthorizationModel - Read the current authorization model
+   * @param options
+   */
+  async readAuthorizationModel(options: ClientRequestOptsWithAuthZModelId = {}): PromiseResult<ReadAuthorizationModelResponse> {
+    const authorizationModelId = this.getAuthorizationModelId(options);
+    if (!authorizationModelId) {
+      throw new Error("authorization_model_id_required");
+    }
+    return this.api.readAuthorizationModel(authorizationModelId, options);
+  }
+
+  /**
+   * ReadLatestAuthorizationModel - Read the latest authorization model for the current store
+   * @param options
+   */
+  async readLatestAuthorizationModel(options: ClientRequestOpts = {}): PromiseResult<ReadAuthorizationModelResponse> {
+    const { headers = {} } = options;
+    setHeaderIfNotSet(headers, CLIENT_METHOD_HEADER, "ReadLatestAuthorizationModel");
+    const authorizationModelsResponse = await this.readAuthorizationModels({ ...options, headers });
+    const response = authorizationModelsResponse as any as CallResult<ReadAuthorizationModelResponse>;
+    response.authorization_model = authorizationModelsResponse.authorization_models?.[0];
+    delete (response as any).authorization_models;
+    return response;
+  }
+
+  /***********************
+   * Relationship Tuples *
+   ***********************/
+
+  /**
+   * Read Changes - Read the list of historical relationship tuple writes and deletes
+   * @param clientReadChangesRequest
+   * @param {object} paginationOptions
+   * @param {number} [paginationOptions.pageSize]
+   * @param {string} [paginationOptions.continuationToken]
+   * @param options
+   */
+  async readChanges(clientReadChangesRequest: ClientReadChangesRequest, options: ClientRequestOptsWithAuthZModelId & PaginationOptions = {}): PromiseResult<ReadChangesResponse> {
+    return this.api.readChanges(clientReadChangesRequest.type, options.pageSize, options.continuationToken, options);
+  }
+
+  /**
+   * Read - Read tuples previously written to the store (does not evaluate)
+   * @param clientReadRequest
+   * @param options
+   */
+  async read(clientReadRequest: ClientReadRequest, options: ClientRequestOpts = {}): PromiseResult<ReadResponse> {
+    return this.api.read({ tuple_key: clientReadRequest }, options);
+  }
+
+  /**
+   * Write - Create or delete relationship tuples
+   * @param {object} body
+   * @param {objects} options
```

Comment:
ideally, we highlight the major options (such as maxPerChunk) here so that caller does not need to read inside the code.

Reply:
Done